From 4e1a8c932df72cc0c7703c3af1e900b7a69bf238 Mon Sep 17 00:00:00 2001
From: Lionel Debieve <lionel.debieve@foss.st.com>
Date: Fri, 7 Apr 2023 14:42:13 +0200
Subject: [PATCH 373/954] fix(st): fix the ssp_otp nvmem for stm32mp13

NOT_UPSTREAMABLE

STM32MP13 use the cfg9_otp instead of ssp_otp in the
associated device tree.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 305ef3d59e99718c482970d5e9828ab626b5ab9c fix(st): fix the ssp_otp nvmem for stm32mp13]
Signed-off-by: Lionel Debieve <lionel.debieve@foss.st.com>
Change-Id: I5f9861422209c60931b78574045e62999c9f66d7
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/299185
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
---
 plat/st/stm32mp1/stm32mp1_def.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/plat/st/stm32mp1/stm32mp1_def.h b/plat/st/stm32mp1/stm32mp1_def.h
index 826e564a51..a12bbcdf74 100644
--- a/plat/st/stm32mp1/stm32mp1_def.h
+++ b/plat/st/stm32mp1/stm32mp1_def.h
@@ -438,9 +438,11 @@ enum ddr_type {
 #if STM32MP13
 #define NAND_OTP			"cfg9_otp"
 #define NAND2_OTP			"cfg10_otp"
+#define SSP_OTP				"cfg9_otp"
 #endif
 #if STM32MP15
 #define NAND_OTP			"nand_otp"
+#define SSP_OTP				"ssp_otp"
 #endif
 #define MONOTONIC_OTP			"monotonic_otp"
 #define UID_OTP				"uid_otp"
@@ -448,7 +450,6 @@ enum ddr_type {
 #define ENCKEY_OTP			"oem_enc_key"
 #define BOARD_ID_OTP			"board_id"
 #define CFG2_OTP			"cfg2_otp"
-#define SSP_OTP				"ssp_otp"
 #define CHIP_CERTIFICATE_OTP		"chip_otp"
 #define RMA_OTP				"rma_otp"
 
-- 
2.53.0

